index.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Tech5</title>
+  <link rel="apple-touch-icon" sizes="180x180" href="/images/favicon/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon/favicon-16x16.png">
+  <link rel="manifest" href="/images/favicon/site.webmanifest">
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <header>
+    <section class="header__container">
+      <nav class="header__menu_bar">
```

Comment:
If `menu_bar` in `header__menu_bar` it's a thing, best to use hyphens for that one thing. E.g. `menu-bar`

Reply:
I think it might be worth for me to invest a bit of time to learn a convention. Maybe BEM? What would you suggest?

Reply:
BEM is fine